cypress/tests/status-card.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { NO_WAIT, STATUS_CARD_TITLE } from '../constants/common';
+import { MCG_MS_Common } from '../views/mcg-ms-common';
+
+describe('admin dashboard - status card', () => {
+  before(() => {
+    cy.login();
+  });
+
+  after(() => {
+    cy.logout();
+  });
+
+  beforeEach(() => {
+    MCG_MS_Common.visitMcgMsDashboard();
+  });
+
+  it('status should be healthy and not contain health state message', () => {
+    cy.get(`[data-test="${STATUS_CARD_TITLE}-health-item"]`).then(($item) => {
+      cy.wrap($item)
```

Comment:
why do we need wait for `$item` to get resolved ?
we are already inside `.then()` block, right... it means it is resolved...

Reply:
we can directly use `byTestID` as well...

Reply:
why I added then is because after getting the particular HTML element I want to query within it

Reply:
I am not asking why u added then... I can saying 
`why do we need wait for $item to get resolved ?
we are already inside .then() block, right... it means it is resolved...`

Reply:
no need for `wrap`

Reply:
the content within then is an HTML element and we can't have cypress queries on them.

Reply:
that is why we have byTestID